stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channel/info/ChannelInfo.kt

```diff
@@ -51,58 +46,63 @@ import io.getstream.chat.android.compose.state.channel.list.LeaveGroup
 import io.getstream.chat.android.compose.state.channel.list.ViewInfo
 import io.getstream.chat.android.compose.ui.common.avatar.UserAvatar
 import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.chat.android.compose.ui.util.getLastSeenText
 import io.getstream.chat.android.compose.ui.util.isDistinct
 
 /**
  * Shows special UI when an item is selected.
  * It also prepares the available options for the channel, based on if we're an admin or not.
  *
  * @param selectedChannel The channel the user selected.
- * @param user Currently logged-in user data.
+ * @param currentUser The currently logged-in user data.
  * @param onChannelOptionClick Handler for when the user selects a channel option.
  * @param modifier Modifier for styling.
  * @param shape The shape of the component.
  */
 @Composable
 public fun ChannelInfo(
```

Comment:
I'm curious about the reason we are using `Box` instead of `BottomSheetScaffold` to show up `ChannelInfo` on the `ChannelsScreen`.

Reply:
Yes, also curious. We need to review attachment picker dialog and channel info dialog to see if we can use `BottomSheetScaffold`.

Reply:
Generally - I'm trying to avoid using things like the BottomSheets and special types of Scaffolds etc, because I feel like it's very contraining.

With a Box that fills the entire screen as an overlay, as well as the ChannelInfo component (internal content), our users can choose to use modifiers to position the sheet in the center of the screen, at the top or bottom etc.

It's not constraining, while the BottomSheet is and it entails more complexity to the screen than a simple Box/Card/Surface combination.

For the Screen type of components, it doesn't matter much what we use I guess, but if we make the ChannelInfo a BottomSheet and require it to be used within a Scaffold or something, then we'll potentially have limitations for our users.

Within the Screen components however, we could try using higher-level components to make it easier, but we'd have to do it in a way that doesn't force our users to use a specific pattern!